modules/nf-core/shapeit5/phasecommon/main.nf

```diff
@@ -0,0 +1,51 @@
+process SHAPEIT5_PHASECOMMON {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::shapeit5=1.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/shapeit5:1.0.0--h0c8ee15_0':
+        'quay.io/biocontainers/shapeit5:1.0.0--h0c8ee15_0'}"
+
+    input:
+        tuple val(meta) , path(input), path(input_index), val(region), path(pedigree)
```

Comment:
```suggestion
        tuple val(meta) , path(input), path(input_index), path(pedigree)
```

Since the region is an optional input, this should be given through ext.args

Reply:
Sorry region parameter is mandatory I made the change accordingly.